spec/spec_helper.rb

```diff
@@ -80,6 +90,22 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.before type: :system do
+    # Disable UI system tests by default when testing locally
+    skip('UI system testing is currently disabled') unless ENV.fetch('ENABLE_UI_TESTING', nil) == 'true'
```

Comment:
An environment variable isn't a very elegant way of handling this. Since these tests are different, we can expect developers to run them separately from the main test suite.

You can configure rspec to ignore some paths by default, adding to a `.rspec` config file. See https://relishapp.com/rspec/rspec-core/v/3-11/docs/configuration/pattern or https://relishapp.com/rspec/rspec-core/v/3-11/docs/configuration/exclude-pattern (you'll need to experiment). On GitHub actions run two separate sets of tests, one with just the default `rspec` (everything except the system tests), and then one for the system tests.